exdb/templates/exdb/experience_approval.html

```diff
@@ -11,66 +11,17 @@
         <h1>{% trans "Experience Approval" %}</h1>
         <p><a href="{% url 'hallstaff_dash' %}">Return to Hall Staff Dashboard</a></p>
```

Comment:
I realize you didn't create this... but since you're editing this file anyway, would you mind wrapping this in a `{% trans %}` tag too?